packages/legend-graph/src/graph-manager/action/changeDetection/DSL_Mapping_ObserverHelper.ts

```diff
@@ -733,14 +734,17 @@ export const observe_SubstituteStore = skipObserved(
 
 export const observe_MappingInclude = skipObserved(
   (metamodel: MappingInclude): MappingInclude => {
-    makeObservable(metamodel, {
-      included: observable,
-      storeSubstitutions: observable,
-    });
+    if (metamodel instanceof MappingIncludeMapping) {
+      makeObservable(metamodel, {
+        included: observable,
+        storeSubstitutions: observable,
+      });
 
-    observe_PackageableElementReference(metamodel.included);
-    metamodel.storeSubstitutions.forEach(observe_SubstituteStore);
+      observe_PackageableElementReference(metamodel.included);
+      metamodel.storeSubstitutions.forEach(observe_SubstituteStore);
 
+      return metamodel;
```

Comment:
lets add a small TODO comment here on adding extension mechanism here for when we add form editor for inlcudes mapping.

Reply:
Added